tools/expotools/src/commands/UpdateVendoredModule.ts

```diff
@@ -678,6 +701,29 @@ async function action(options: ActionOptions) {
     return bundledNativeModules;
   });
 
+  const { name, version } = (await JsonFile.readAsync(path.join(tmpDir, 'package.json'))) as {
+    name: string;
+    version: string;
+  };
+  const semverPrefix = (options.semverPrefix != null ? options.semverPrefix : moduleConfig.semverPrefix) || '';
```

Comment:
If you need this variable here, I wouldn't duplicate it. Let's define it a few lines before and remove it from `if` checking `moduleConfig.installableInManagedApps`. Probably we could do the same with `versionRange`, you use it below as `${semverPrefix}${version}`